Sources/Models/MessageData.swift

```diff
@@ -62,4 +62,67 @@ public enum MessageData {
 //    
 //    case placeholder
 
+    // MARK: - Equatable
+
+    public static func ==(lhs: MessageData, rhs: MessageData) -> Bool {
+        switch (lhs, rhs) {
+        case (let .text(string1), let .text(string2)):
+            return true
+
+        case (let .attributedText(string1), let .attributedText(string2)):
```

Comment:
For example, this is a very light check for equality. Ideally, we would have to compare the contents of each String. But I really want to remove these conformances all together 😅